Triplet vcpkg build instruction update (#5656)
PointCloudLibrary/pcl: doc/tutorials/content/pcl_vcpkg_windows.rst

  ./vcpkg install pcl
  
.. note::
  This will build executables 2 times in release mode, as default host triplet is x64-windows

>   in various executables, but we use x86 by default. So to fix it you can provide default

Reviewer: ```suggestion
  on most modern PC systems, but vcpkg install x86 by default. So to fix it you can set
```

Author: Updated

  host-triplet same as default triplet.

>     ./vckpgk install pcl --host-triplet -x86 

Reviewer: Also don't you need to specify the full x86-windows ?

>     ./vckpgk install pcl --host-triplet -x86 

Author: Yeah sure, it should be `x86-windows` without dash

>     ./vckpgk install pcl --host-triplet x86-windows 

Reviewer 2: ```suggestion
    ./vcpkg install pcl --host-triplet x86-windows 
```

  Or, you can use same custom triplet for both --triplet and --host-triplet

>     ./vckpgk install pcl --triplet <same_custom_triplet_type> --host-triplet <same_custom_triplet_type>

Reviewer 2: ```suggestion
    ./vcpkg install pcl --triplet <same_custom_triplet_type> --host-triplet <same_custom_triplet_type>
```
